build-system/server.js

```diff
@@ -431,7 +431,7 @@ app.use(['/examples/*', '/extensions/*'], function (req, res, next) {
   next();
 });
 
-app.get('/examples/*', function(req, res, next) {
+app.get(['/examples/*', '/test/manual/*'], function(req, res, next) {
```

Comment:
I see.  so we never need to replace script path from cdn to local path in `/test/manual` in the future? That's great!